packages/dashboard/src/components/pageStructure/index.js

```diff
@@ -59,6 +69,23 @@ import {
   PathName,
 } from './navigationComponents';
 
+const NEW_TEMPLATE_THRESHOLD_IN_DAYS = 60;
+
+function getNewSlugs(metaDataEntries, days) {
+  const currentDate = toDate(new Date(), getOptions());
+  return metaDataEntries
+    .map((metaData) => ({
+      ...metaData,
+      creationDate: toDate(metaData.creationDate, getOptions()),
+    }))
+    .map((metaData) => [
+      metaData.slug,
+      differenceInDays(currentDate, metaData.creationDate),
+    ])
+    .filter(([, deltaDays]) => deltaDays < days)
+    .map(([slug]) => slug);
+}
```

Comment:
Nit: There's no need for the second `map()`. Plus, returning an array from there is a bit confusing.

```suggestion
function getNewSlugs(metaDataEntries, days) {
  const currentDate = toDate(new Date(), getOptions());
  return metaDataEntries
    .map((metaData) => ({
      ...metaData,
      creationDate: toDate(metaData.creationDate, getOptions()),
    }))
    .filter(({creationDate}) => differenceInDays(currentDate, creationDate) < days )
    .map({slug}) => slug);
}
```

Or could even do without the first map:


```suggestion
function getNewSlugs(metaDataEntries, days) {
  const currentDate = toDate(new Date(), getOptions());
  return metaDataEntries
    .filter(({creationDate}) => differenceInDays(currentDate, toDate(creationDate, getOptions())) < days )
    .map({slug}) => slug);
}
```